stores/dotenv/store.go

```diff
@@ -81,7 +81,7 @@ func (store *Store) LoadPlainFile(in []byte) (sops.TreeBranches, error) {
 			}
 			branch = append(branch, sops.TreeItem{
 				Key:   string(line[:pos]),
-				Value: strings.Replace(string(line[pos+1:]), "\\n", "\n", -1),
+				Value: string(line[pos+1:]),
```

Comment:
The changes to this file don't seem to belong in this PR

Reply:
It might have been a mistake, I'll undo all changes in `stores/dotenv/store.go`